### ISSUE TITLE
the loss function

### ISSUE THREAD

**Issue by guzh970630** [Delver-of-Squeakrets/LISNN#1]
hi, I ask a question. Why use the MSE loss function for the loss function. It is well known that it is better to use the cross-entropy loss function for classification problems, and MSE should be used for regression problems. But why is the MSE loss function used in the end?

**Comment by Delver-of-Squeakrets** [comment]
Using MSE loss function is mainly for the character of SNNs and the rate coding we used in this work.  It's more like fitting output trains for teacher trains  rather than a traditional classification task.
